google/cloud/pubsub/internal/tracing_batch_callback.cc

```diff
@@ -192,6 +192,49 @@ class TracingBatchCallback : public BatchCallback {
     }
   }
 
+  Span StartModackSpan(
+      google::pubsub::v1::ModifyAckDeadlineRequest const& request) override {
+    namespace sc = opentelemetry::trace::SemanticConventions;
+    using Attributes =
+        std::vector<std::pair<opentelemetry::nostd::string_view,
+                              opentelemetry::common::AttributeValue>>;
+    using Links =
+        std::vector<std::pair<opentelemetry::trace::SpanContext, Attributes>>;
+
+    if (request.ack_ids().empty()) return {};
```

Comment:
One final question:  Is this a feature?  That is, why not just proceed with the empty ack_ids?

Reply:
Excellent.  And with that, it looks like the `if (span.span)` guard in `EndModackSpan()` can also disappear.